### ISSUE TITLE
Video Tutorial Feature Request

### ISSUE THREAD

**Issue by MSKirk** [Helioviewer-Project/helioviewer.org#199]
As more features and capabilities are added into Helioviewer (which are fantastic), it is becoming more challenging for a novice user to discover all of these on their own. A short (youtube?) video guide highlighting how to make use of each of the features would help significantly.

**Comment by wafels** [comment]
Good idea.  Version 3.3 will add in several new features that expand the usefulness of helioviewer.org.

**Comment by wafels** [comment]
See #201.
